Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.cpp

```diff
@@ -193,10 +193,26 @@ namespace AzToolsFramework
         return m_systemCursorState;
     }
 
+    static constexpr AZ::u32 SyntheticDeviceOffset = 1000;
```

Comment:
this constant is the same constant in ViewportControllerList.cpp?

Reply:
Aah good point I'll double check 👍

Reply:
Hey @pollend, I had a look in `ViewportControllerList.h/cpp` but couldn't see any mention of this constant. Currently we do just use this in `QtEventToAzInputMapper.cpp` so I'm going to leave as-is for now. Thanks!